src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -284,14 +182,22 @@
         <!-- Bucket -->
         <div class="form-group row"
              *ngIf="nfsForm.getValue('name') === 'RGW'">
-          <label class="cd-col-form-label required"
-                 for="path"
-                 i18n>Path</label>
+          <label class="cd-col-form-label"
+                 for="path">
+            <span class="required"
+                  i18n>Path (Bucket)</span>
+            <cd-helper>
+              <p i18n>The path must be an existing bucket. If multiple realms are configured, the bucket must belong to the default realm.</p>
+              <span i18n>You can create a new one <a routerLink="/rgw/bucket/create"
+                                                     class="btn-link">here</a>.</span>
```

Comment:
Same thing here (["description-reason-resolution"](https://www.patternfly.org/v4/ux-writing/error-messages/)):
* **Info message**: unnecessary if the field is named "bucket".
* **Validation error**: "The bucket doesn't exist or is not in the default realm (if multiple realms are configured). To continue, [create a new bucket]()".